src/main/java/gui/ChatterboxResponses.java

```diff
@@ -0,0 +1,128 @@
+package gui;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import tasks.Task;
+
+/**
+ * Handles the string formatting of chatterbox responses for the gui
+ */
+public class ChatterboxResponses {
+    private static final String LINE_SEPARATOR = "____________________________________________________________";
+    private static final String BOTNAME = "Chatterbox";
+
+    private static final DateTimeFormatter PRINTDATEFORMATTER = DateTimeFormatter.ofPattern("MMM dd yyyy, HH:mm");
```

Comment:
Would be better if <code>BOTNAME</code> changed to <code>BOT_NAME</code> and <code>PRINTDATEFORMATTER</code> to <code>PRINT_DATE_FORMATTER</code> for consistency